scripts/restore

```diff
@@ -70,43 +73,59 @@ ynh_system_user_create --username="$app"
 #=================================================
 
 # Restore permissions on app files
-#chown -R root: $final_path
-chown -R jellyfin:jellyfin "$plugins_path"
+chown -R $app: $final_path
+chown -R $app: /var/lib/jellyfin
```

Comment:
```suggestion
chown -R $app: /var/lib/$app
```

Reply:
Same here, I would prefer to leave it explicitly to `/var/lib/jellyfin` since we have no choice.